packages/graphql-cli-validate-magento-pwa-queries/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "graphql-cli-validate-magento-pwa-queries",
+  "version": "1.0.0",
+  "description": "graphql-cli plugin for validating all queries in a Magento 2 PWA project",
+  "main": "index.js",
+  "scripts": {
+    "build": "echo 'Skipping graphql-cli-validate-magento-pwa-queries build...'",
+    "clean": "rimraf ./node_modules",
+    "test": "jest"
+  },
+  "repository": "https://github.com/magento-research/pwa-studio",
+  "keywords": [
+    "graphql",
+    "graphql-cli",
+    "plugin"
+  ],
+  "author": "Adobe, Inc",
+  "license": "MIT",
```

Comment:
Are these correct? Typically we use the following:

```
"author": "Magento Commerce",
"license": "(OSL-3.0 OR AFL-3.0)",
```

Reply:
Good question, I asked about this too and @zetlen said this was ok.

Reply:
Yeah, we can do either.